vibes/vibed-ui/src/core/Event.cpp

```
@@ -0,0 +1,46 @@
+// MIT License
+// Copyright (c) 2026 Palaash
+
+#include "Event.h"
+
+#include <vector>
+
+namespace {
+
+std::vector<core::Event> g_eventQueue;
```

Clang (14.0.6)

[warning] variable 'g_eventQueue' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)